Cargo.toml

```
@@ -147,6 +147,10 @@ goauth = { version = "0.10.0", default-features = false, optional = true }
 gouth = { version = "0.2.1", default-features = false, optional = true }
 smpl_jwt = { version = "0.6.1", default-features = false, optional = true }
 
+# Amqp
+lapin = { version = "1.6", optional = true }
+tokio-amqp = { version = "1.0", optional = true }
```

```suggestion
lapin = { version = "1.6", default-features = false, optional = true }
tokio-amqp = { version = "1.0", default-features = false, optional = true }
```

We've started to get in the habit of turning off default features and just including the ones that we need. You might need to add a `features = []` option to both of these with any needed features that we were previously getting by default.